js/bitmart.js

```diff
@@ -715,31 +715,31 @@ module.exports = class bitmart extends Exchange {
         const marketId = this.safeString2 (ticker, 'symbol', 'contract_id');
         market = this.safeMarket (marketId, market);
         const symbol = market['symbol'];
-        const last = this.safeNumber2 (ticker, 'close_24h', 'last_price');
-        let percentage = this.safeNumber2 (ticker, 'fluctuation', 'rise_fall_rate');
+        const last = this.safeString2 (ticker, 'close_24h', 'last_price');
+        let percentage = this.safeString2 (ticker, 'fluctuation', 'rise_fall_rate');
         if (percentage !== undefined) {
             percentage *= 100;
```

Comment:
We cannot multiply a string by one hundred here, we should use Precise string-based maths as explained here:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings